schemas/lst/schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "schema": "Liquid Staking Tokens (LSTs)",
+    "description": "Standard table definitions for LSTs (liquid staking tokens)",
+    "version": "1.0.0-alpha",
+    "tables": [
+      {
+        "label": "Position Snapshot",
+        "tableName": "lst_holders",
+        "aggregation": "daily",
+        "description": "User level snapshot of holders in this protocol.",
+        "properties": {
+          "timestamp": {
+            "description": "The timestamp of the snapshot.",
+            "type": "number"
+          },
+          "protocolName": {
+            "description": "The name of the protocol.",
+            "type": "string"
+          },
+          "chainId": {
+            "description": "Standard chain ID.",
+            "type": "number"
+          },
+          "tokenAddress": {
+            "description": "The address of the LST in the protocol.",
+            "type": "string"
+          },
+          "tokenSymbol": {
+            "description": "The symbol of the token.",
+            "type": "string"
+          },
+          "userAddress": {
+            "description": "The address of the user holding the LST",
+            "type": "string"
+          },
+          "amount": {
+            "description": "The amount of LST tokens held in the protocol, decimal normalized."
+          },
+          "amountUsd": {
+            "description": "The amount held in USD.",
+            "type": "number"
+          }
+        }
+      },
+      {
+        "label": "Protocol Snapshot",
+        "tableName": "lst_metrics",
+        "aggregation": "daily",
+        "description": "Snapshot at the protocol level, including, TVL and fees data.",
+        "properties": {
+          "timestamp": {
+            "description": "The timestamp of the snapshot.",
+            "type": "number"
+          },
+          "protocolName": {
+            "description": "The name of the protocol.",
+            "type": "string"
+          },
+          "chainId": {
+            "description": "Standard chain ID.",
+            "type": "number"
+          },
+          "tvlUsd": {
+            "description": "The total value locked in USD (ie, the value of the liquid staking tokens in the protocol).",
+            "type": "number"
+          },
+          "feesUsd": {
+            "description": "The fees collected in USD in the given snapshot period.",
+            "type": "number"
```

Comment:
these will be in the schema.md too?

`fees_usd` probably optional here too

Reply:
The schema.md is the source of truth to review here. Remember the conversation about removing the protocol snapshot, that is why it is not in there

Reply:
im fine with or without this actually, just that we need to sum up all holders = protocol tvl.

and if its too slow we have to materialize it, otherwise it should be fine without

Reply:
Actually, I think it makes sense to keep for LSTs. That way we could have fees